src/test-utils/dom/tabs/index.ts

```diff
@@ -22,20 +23,69 @@ export default class TabsWrapper extends ComponentWrapper<HTMLButtonElement> {
     return this.find(`.${styles['tabs-tab']}:nth-child(${index}) .${styles['tabs-tab-link']}`);
   }
 
+  /**
+   * Finds the tab header container at the given position (1-based) and returns the element.
+   *
+   * @param index 1-based index of the clickable element to return
+   */
+  findTabHeaderContentByIndex(index: number): ElementWrapper<HTMLAnchorElement | HTMLButtonElement> | null {
+    return this.find(`.${styles['tabs-tab']}:nth-child(${index}) .${styles['tabs-tab-header-container']}`);
+  }
+
   /**
    * Finds the tab with the given ID and returns the clickable element from its tab label.
    *
-   * @param index ID of the clickable element to return
+   * @param id ID of the clickable element to return
    */
   findTabLinkById(id: string): ElementWrapper<HTMLAnchorElement | HTMLButtonElement> | null {
     return this.find(`.${styles['tabs-tab-link']}[data-testid="${id}"]`);
   }
 
+  /**
+   * Finds the dimissible button by using the tab index
```

Comment:
let's standardize on either "close" or "dismiss" button across here and the component API docs